src/pages/Search/EmptySearchView.tsx

```diff
@@ -1,43 +1,147 @@
-import React, {useMemo} from 'react';
+import {Str} from 'expensify-common';
+import React, {useCallback, useMemo, useState} from 'react';
+import {Linking, View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import DotIndicatorMessage from '@components/DotIndicatorMessage';
 import EmptyStateComponent from '@components/EmptyStateComponent';
+import type {FeatureListItem} from '@components/FeatureList';
 import * as Illustrations from '@components/Icon/Illustrations';
+import LottieAnimation from '@components/LottieAnimations';
+import MenuItem from '@components/MenuItem';
 import SearchRowSkeleton from '@components/Skeletons/SearchRowSkeleton';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
 import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
 import Navigation from '@libs/Navigation/Navigation';
 import variables from '@styles/variables';
+import * as Link from '@userActions/Link';
 import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
 import type {SearchDataTypes} from '@src/types/onyx/SearchResults';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
 
 type EmptySearchViewProps = {
     type: SearchDataTypes;
 };
 
+const tripsFeatures: FeatureListItem[] = [
+    {
+        icon: Illustrations.PiggyBank,
+        translationKey: 'travel.features.saveMoney',
+    },
+    {
+        icon: Illustrations.Alert,
+        translationKey: 'travel.features.alerts',
+    },
+];
+
 function EmptySearchView({type}: EmptySearchViewProps) {
     const theme = useTheme();
     const StyleUtils = useStyleUtils();
     const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const [travelSettings] = useOnyx(ONYXKEYS.NVP_TRAVEL_SETTINGS);
+    const [activePolicyID] = useOnyx(ONYXKEYS.NVP_ACTIVE_POLICY_ID);
+    const [primaryLogin = ''] = useOnyx(ONYXKEYS.ACCOUNT, {selector: (account) => account?.primaryLogin});
+    const policy = usePolicy(activePolicyID);
+    const [ctaErrorMessage, setCtaErrorMessage] = useState('');
+
+    const hasAcceptedTravelTerms = travelSettings?.hasAcceptedTerms;
+    const hasPolicyAddress = !isEmptyObject(policy?.address);
+
+    const onBookATripPress = useCallback(() => {
```

Comment:
This code comes from ManageTrips, right? If yes, let's create a Util function to DRY code